### PR TITLE
Ports the payday meme alongside the associated vendor

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request

StrongDMM_FvH12OHWqk

dreamseeker_E6bqMvPFs6

## Why It's Good For The Game

haha funny meme

## Changelog
```changelog Toriate
add: Ported payday meme from upstream together with the trash vendor
```

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1471,6 +1471,58 @@
 	vendor_department = DEPARTMENT_CIVILIAN
 */
 
+/obj/machinery/vending/Billomat
+	name = "Bill Trustworthy's Discount Guns and Enterprising Detritus"
+	desc = "Some relic of an arms dealer's business, its owner most likely long dead."
+	product_slogans = "Discount guns for discount prices!;Also see our used ship line!;From the home of Challenge Pissing!"
```
Comment (Jamini) [inline code/game/machinery/vending.dm:1477]: "From the home of Challenge Pissing!""

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1471,6 +1471,58 @@
...
+	contraband = list(/obj/item/weapon/gun/projectile/mandella = 4,/obj/item/ammo_magazine/cspistol = 12)
+	prices = list(
+					/obj/item/ammo_magazine/lrifle = 400,
+					/obj/item/ammo_magazine/hpistol = 300,
+					/obj/item/ammo_magazine/cspistol = 400,
+					/obj/item/ammo_magazine/srifle = 300,
+					/obj/item/ammo_magazine/smg = 400,
+					/obj/item/part/armor = 700,
+					/obj/item/part/gun = 700,
+					/obj/item/weapon/gun/projectile/automatic/ak47/fs = 3200,
```
Comment (Jamini) [inline code/game/machinery/vending.dm:1509]: Guns of this tier probably shouldn't be available from a vendor. We have a trade station for these.

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1471,6 +1471,58 @@
...
+					/obj/item/weapon/gun/projectile/automatic/z8 = 3500,
+					/obj/item/weapon/gun/projectile/automatic/molly = 2000,
+					/obj/item/weapon/gun/projectile/shotgun/pump/gladstone = 2200,
+					/obj/item/weapon/storage/deferred/crate/clown_crime = 1800,
+					/obj/item/weapon/storage/deferred/crate/clown_crime/wolf = 1800,
+					/obj/item/weapon/storage/deferred/crate/clown_crime/hoxton = 1800,
+					/obj/item/weapon/storage/deferred/crate/clown_crime/chains = 1800
+					)
+	idle_power_usage = 211
+	vendor_department = DEPARTMENT_CIVILIAN
```
Comment (Jamini) [inline code/game/machinery/vending.dm:1524]: Wrong department. This should be cargo.

File: maps/CEVEris/_CEV_Eris.dmm [hunk 4]
```diff
@@ -52109,7 +52109,7 @@
 /turf/simulated/floor/tiled/white,
 /area/eris/rnd/mixing)
 "crj" = (
-/obj/machinery/vending/powermat,
+/obj/machinery/vending/Billomat,
```
Comment (Jamini) [inline maps/CEVEris/_CEV_Eris.dmm:52112]: This shouldn't be there.

**Review by Jamini — CHANGES_REQUESTED** [review]
Lots of issues with the content of this PR.

**Comment by Michiyamenotehifunana** [comment]
Vendor is no longer mapped in, now the new content has been moved to the ammo vendors.

**Comment by Michiyamenotehifunana** [comment]
Now contraband in the ammo vendor

**Review by Jamini — APPROVED** [review]

**Comment by ShadzFen** [comment]
I do really wish you'd give these PRs more effort. Anyone reading this PR will have no clue what's being added other than a bag, a payday mask and a vendor uwu

**Pull request merged**